.github/workflows/build_test.yml

```diff
@@ -0,0 +1,41 @@
+name: Build Test
+on:
+  push:
+    branches:
+    - 'melodic-devel'
+  pull_request:
+    branches:
+    - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Ubuntu 18.04?

Reply:
That is just the environment the actions will run. I am running a container for nelodic on the next line